### ISSUE TITLE
Maximum Open Tickets not enforced for staff

### ISSUE THREAD

**Issue by ntozier** [osTicket/osTicket-1.7#545]
Via Web interface:

If a staff opens a ticket for someone else's email address and that email address already has reached the maximum open tickets, a warning email is generated but it opens the ticket anyway.

Email message reads:

"Max open tickets (5) reached  for fake@email.com

http://support.domain.ext/scp/tickets.php?a=open"

Shouldn't the Admin panel -> Settings -> Email -> Maximum Open Tickets actually generate an error and NOT open a ticket?  Or at least have a "Staff can override ticket limit" option some where?

If a client tries to open a ticket beyond the limit it generates an onscreen warning, and sends a notification to the admin. Side thought: Shouldn't it also send an email to the client explaining that they have reached their limit of allowed open tickets, and that they should contact support directly if they need to open more or something?  Or perhaps have it be a toggle setting so that admins get to choose if the client is also emailed.

**Comment by protich** [comment]
- Staff are exempted from the limit. 
  My rationale was; a new ticket might need to be created (e.g in a separate dept.) to resolve an issue blocking the closure of the already open tickets.
- The "over limit notice" is sent to client - when enable - but only once to avoid loops (when the limit is first reached). 
  Settings to disable/enable is in Admin Panel > Settings > Autoresponder. 
- The warning to admin email is expected - default behavior on all denied tickets.
